### DISCUSSION TITLE
Enable auto hibernate or sleep

### DISCUSSION THREAD

**Discussion by Itz-Abhishek-Tiwari** [prasanthrangan/hyprdots#1223] [Q&A]
i am using hyprland with your dot files and liking it too much where i can switch back to any DE i just want to know how would i able to enable sleep if my laptop is turn on for more then 30 min.

**Comment by kRHYME7** [comment]
Here you go https://github.com/prasanthrangan/hyprdots/pull/1173

**Reply by Itz-Abhishek-Tiwari** [reply]
"  🕛 "  i am not able to view this because of font is missing can you tell me what is this

**Reply by kRHYME7** [reply]
Ow that's the clock glyph. 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/e76dead4-31db-4576-b022-23d5f88848c8)


What I meant with that part is some apps won't respect the idle inhibitors so users can still manually inhibit idle manually using that . ( it's already available )

**Reply by Enigma1309** [reply]
ttf-jetbrains-mono
ttf-jetbrains-mono-nerd
